CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - equipment specific settings can be found in the 'edit equipment' menu
 - Added icon to the magneto stick
 - Added the function `AddToSettingsMenu` to both `SWEP` and `ITEM` to add settings to the equipment menu
+- Added the role flag `.isOmniscientRole`; if set to true the role is able to see missing in action players and the haste mode time
```

Comment:
```suggestion
- Added the role flag `.isOmniscientRole`, if set to true the role is able to see missing in action players and the haste mode time
```

Reply:
I did not see this when merging. However I think that a `;` is more fitting anyway

Reply:
well, we nver use semicolons, but it's okay 😄

Reply:
I actually use them quite a lot in in-code comments